rpa-manager/modules/ROOT/pages/processautomation-prepare-project-application.adoc

```diff
@@ -0,0 +1,111 @@
+
+== Create a Process Application
+
+Create one entry for each application instance required for the execution of processes.
+
+To create a process application:
+
+. Open the *Process Applications* view of the *Process Automation* module.
+. Click *Create*.
+. [[form-create-process-applications]] Complete the *Create a new application* form:
++
+* *Time Zone*:
++
+Time zone for defining downtimes.
++
+Specify a downtime in the time zone in which it occurs. If the bot that you assign the downtime to is installed on a computer that uses a different time zone, the downtime interval is converted to the bot’s time zone. If you assign the application to multiple bots in different time zones, all bots pause executing processes that use that application simultaneously. The time intervals might change, depending on daylight saving time.
++
+For example, if you specify a downtime that starts at 1:30 AM and ends at 5:30 AM in Tokyo time (JST (UTC +9)) and the bot is installed on a computer in San Francisco (PST (UTC -8)), the bot does not execute processes between 8:30 AM and 12:30 PM in San Francisco time.
++
+To avoid confusion, append the name of the time zone to the name of the downtime. For example, name your downtime `Electrical Power Outage Tokyo`.
++
+* *Downtimes*
++
+Click *Create* to add a downtime to the application.
+. Click *Save*.
+
+== Edit an Application
+
+Edit an application to change its data or down-times:
+
+. Open the *Process Applications* view of the *Process Automation* module.
+. Click *Create*.
+. Edit the *Edit the application* form.
++
+For an explanation of the properties, see  <<form-create-process-applications, *Create an Application*>>.
+. Click *Save*.
+
+== Deactivate or Activate a Process Application
+
+Deactivate an application if it is temporarily not available and you don't know when it will be available again.
+
+Use down-times to schedule maintenance and planned downtimes. Deactivation periods are not added to the list of down-times.
+
+Applications are not available during down-times or while deactivated. A member of the process team can deploy a process on a bot that has assigned the required processes. If an application becomes non-available, the bot completes running processes using that application and restarts them when the downtime or deactivation period has ended.
+
+If an application is not available, deactivate it:
+
+. Open the *Process Applications* view of the *Process Automation* module.
+. Click the *Deactivate* icon image:deactivate-icon.png[toggle-on symbol,1.5%,1.5%] on the panel of the application you want to deactivate.
+
+RPA Manager displays an *Inactive* label beneath the logo of the application.
+
+If a deactivated application becomes available again, activate it:
+
+. Open the *Process Applications* view of the *Process Automation* module.
+. Click the *Activate* icon image:activate-icon.png[toggle-off symbol,1.5%,1.5%] on the panel of the application you want to activate.
```

Comment:
Global: remove "you want"

Reply:
Removed all that I easily could. Added [W-12942071](https://gus.lightning.force.com/a07EE00001OLu4uYAD) to the backlog for checking the rest...

Reply:
No worries. It's a niggly thing, not an error per se.